buffer.go

```diff
@@ -189,6 +189,73 @@ func (b *Buffer) SwapCharactersBeforeCursor() {
 	}
 }
 
+// SplitWideLines splits lines, whose width is greater than maxWidth to multiple lines.
+func (b *Buffer) SplitWideLines(maxWidth int) *Buffer {
+	text_lines := b.Document().Lines()
+	processed_text := strings.Builder{}
+
+	currentCursor := 0
```

Comment:
Pls, don't mix camelCase and snake_case in one scope.

Reply:
Fixed.